PayPal/src/test/java/com/braintreepayments/api/paypal/PayPalVaultErrorHandlingEditRequestUnitTest.java

```diff
@@ -0,0 +1,25 @@
+package com.braintreepayments.api.paypal;
+
+import static junit.framework.Assert.assertEquals;
+import static junit.framework.Assert.assertFalse;
+import static junit.framework.Assert.assertNotNull;
+import static junit.framework.Assert.assertNull;
+
+import com.braintreepayments.api.paypal.vaultedit.PayPalVaultErrorHandlingEditRequest;
+
+import org.junit.Test;
+
+public class PayPalVaultErrorHandlingEditRequestUnitTest {
```

Comment:
What do folks think about not having unit tests for standard Kotlin data objects? @tdchow @saperi22

Reply:
I think maybe these are supposed to be more similar to `PayPalVaultRequestUnitTest` and `PayPalCheckoutRequestUnitTest` where we are instead checking that we are building the parameters as expected? It looks like for edit FI we are currently instead doing that in the `PayPalInternalClient` which maybe is not the pattern we want to follow, not sure though.

Reply:
I also agree - we shouldn't need to test the getting/setting of properties on data classes. However if the data class contains any logic, we should have coverage for that.